app/graphql/resolvers/sample_attachments_resolver.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Resolvers
+  # Sample Attachments Resolver
+  class SampleAttachmentsResolver < BaseResolver
+    type [String], null: true
+
+    alias sample object
+
+    def resolve
+      scope = sample
```

Comment:
Same comment as the one for the attachment metadata resolver. You can remove the scope and update the line below to sample.attachments

Reply:
Fixed in https://github.com/phac-nml/irida-next/pull/408/commits/9b6cba135b8c37588c4339188902cd42339a59b2